containers/reasonml/README.md

```diff
@@ -0,0 +1,57 @@
+# ReasonML
+
+## Summary
+
+*Develop [reasonml](https://reasonml.github.io/en/) applications. Includes [fish shell](https://fishshell.com/).*
```

Comment:
This is parsed by VS Code to create the text in the tool.  Could you remove the links (they'll show up as-is in the UI)?  If you want to include them, you could include a description below the metadata.